lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -174,8 +174,15 @@ def event_action_params(event)
         params[:pipeline] = event.sprintf(@pipeline)
       end
 
-     if @parent
-        params[:parent] = event.sprintf(@parent)
+      if @parent
```

Comment:
What's the story here WRT 5.x ES servers?

This change means that we no longer support parent/child in versions below 6.0

There are two paths forward:

1. We maintain separate 5.x and 6.x compat branches in this output using the gemspec to enforce which versions work with which logstash
2. We do version sniffing inside the plugin and support two different code paths for the two different parent/child impls.

Thoughts?

Reply:
is join_field an option in the output? I don't see it here https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html#plugins-outputs-elasticsearch-parent